docker-compose.yml

```diff
@@ -0,0 +1,11 @@
+version: '3.9'
```

Comment:
Could we please use 3.3 for greater compatibility? That's what my client requires (Fedora 32).